pom.xml

```diff
@@ -275,7 +285,7 @@ tools.jackson.core.*;version=${project.version}
     <dependency>
       <groupId>ch.randelshofer</groupId>
       <artifactId>fastdoubleparser</artifactId>
-      <version>0.9.0</version>
+      <version>${fastdoubleparser.version}</version>
```

Comment:
No point in add property just for sake of adding one; will never be overridden or reused.